### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by mkao006**
A simple clean up for deployment.

### CODE REVIEW

File: pipeline/sentiment_scoring/controller.py [hunk 1]
```diff
@@ -1,554 +1,16 @@
...
-
-# VADER SENTIMENT EXTRACTION
-def VADER2(sentence):
-    '''Function to analyse the sentiment of a single sentence
-
-    '''
-    analyzer = SentimentIntensityAnalyzer()
-    sentiment = analyzer.polarity_scores(sentence)
-    return sentiment
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
```
Comment (marcogarieri) [inline pipeline/sentiment_scoring/controller.py]: Neat!

File: requirements.txt [hunk 2]
```diff
@@ -100,6 +100,7 @@ Twisted==17.5.0
 twitter==1.17.1
 Unidecode==0.4.21
 urllib3==1.22
```
Comment (marcogarieri) [inline requirements.txt]: Should we double check all the packages to remove any useless one?

Comment (mkao006) [reply requirements.txt]: yes, good idea. will do that!

Comment (mkao006) [reply requirements.txt]: Just checked, the `twitter` package is the only one that is not required......

**Pull request merged**